content/applications/inventory_and_mrp/inventory/product_management/product_tracking/type.rst

```diff
@@ -0,0 +1,217 @@
+============
+Product type
+============
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+
+Define *product types* in Odoo to determine how the database keeps track of the product.
```

Comment:
I think this intro sentence could use a brief elaboration to better help the reader understand what product types are

Reply:
Done! :D

```suggestion
Define *product types* in Odoo to track products in varying levels of detail.

Classify products as *storable* to track stock counts, allowing users to trigger :doc:`reordering
rules <../product_replenishment/reordering_rules>` for generating purchase orders. *Consumable*
products are assumed to always be in stock, and *service* products are performed and served by the
business.
```